front/src/BottomBar/StandardButton.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+ * Create a standard SRR Button
+ *
+ * Author: Tom Franken, Gijs van der Sanden
+ *
+ */
+
+const StandardButton = props => {
+  const { onClick, enabled } = props;
+  return (
+    <button className={props.className} onClick={onClick} disabled={!enabled} />
+  );
+};
+
+StandardButton.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  enabled: PropTypes.bool.isRequired
+};
```

Comment:
what is the purpose of making raw html button as a separate component?
no application-specific styling, no functionality or do we expect to add it later?